src/components/ResultsView.tsx

```diff
@@ -671,6 +675,33 @@ const ResultsView: typeof window.roamjs.extension.queryBuilder.ResultsView = ({
                     }}
                   />
                 )}
+                 <MenuItem
+                    icon={"clipboard"}
+                    text={"Copy Query"}
+                    onClick={() => {
+                      const getTextFromTreeToPaste = (items: RoamBasicNode[], indentLevel = 0): string => {
+                        const obj = { textValues: "" };
```

Comment:
haha nice, but this is the same as a `let` (mutable values). The immutable approach I would use:

```typescript
return items.map(item => {
   // return string instead of `+=` to textValues
}).join("")
```

Again, this is a subjective code style thing, so no need to adhere strictly